src/current/v25.1/create-changefeed.md

```diff
@@ -23,6 +23,12 @@ The [examples](#examples) on this page provide the foundational syntax of the `C
 
 ### Legacy privilege model
 
+{{site.data.alerts.callout_info}}
+Starting in v22.2, CockroachDB introduces a new [system-level privilege model]({% link {{ page.version.version }}/security-reference/authorization.md %}#supported-privileges) that provides finer control over a user's privilege to work with the database, including creating and managing changefeeds.
+
+There is continued support for the legacy privilege model for changefeeds in v23.1, however it **will be removed** in a future release of CockroachDB. We recommend implementing the [new privilege model](#required-privileges) for all changefeeds.
```

Comment:
also perhaps implied by the above but it seems like this notice referring to previous versions should be backported to those versions since e.g. v24.3 currently says

> To create a changefeed, the user must be a member of the admin role or have the [CREATECHANGEFEED](https://www.cockroachlabs.com/docs/v24.3/create-user#create-a-user-that-can-control-changefeeds) parameter set.

that said i know little about changefeeds and the associated roles but this seems like some very retroactive legislation of something and I don't understand what is happening here